### ISSUE TITLE
WSL1: \\WSL$ not working

### ISSUE THREAD

**Issue by David-Defoort** [microsoft/WSL#4579]
**Please fill out the below information:**
* Your Windows build number: Microsoft Windows [Version 10.0.18362.356] (1903)

**Problems**
1. Explorer shows Ubuntu under `\\wsl$` but when I double click I get:
![image](https://user-images.githubusercontent.com/5627161/64904747-8d13c180-d6ce-11e9-9ed6-79195b35b292.png)
2. inside a cmd, `dir \\wsl$\Ubuntu` does not works ("The system cannot find the path specified.")
3. (in Ubuntu/bash) `explorer.exe .` inside a /mnt point works fine.  Explorer.exe comes up to the expected path location
4. (in Ubuntu/bash) `explorer.exe .` outside a /mnt point does nothing, it does not even bring explorer at all
5. (in Ubuntu/bash) `code` does not work outside /mnt points, which is what bothers me the most. It used to work and I became dependent of it...

**Attempts**
- apt-get update/upgrade no effect
- uninstall/reinstall Ubuntu and Ubuntu-18.04, no effect
- disable WSL, reboot, re-enable, no effect
- verified the registry as [#4027](https://github.com/microsoft/WSL/issues/4027#issuecomment-494969089) said, no effect

**Notes**
It all started after upgrading to 1903... `explorer .` and `code` were working fine before, regardless of the current directory

**Comment by abreingan** [comment]
I have been having the same problem as David-Defoort, except with the Debian distro. 
I started using WSL recently and was never able to use explorer.exe outside mount points. 
I went to 1903 18362.356 and tried everything David tried without a success. 
I am now on 10.0.18362.388 and retried the disable wsl and that was successful (not sure if this is permanent or not). 
To confirm this was via admin cmd prompt and wsl -t Debian.
I also note Hyper-V is active on this machine.
To be very picky (:-)> starting explorer with \\wsl$\Debian and then navigating to /mnt/c fails... "Attempt to access invalid address" OK I was being deliberately difficult - thanks for a very useful product.
